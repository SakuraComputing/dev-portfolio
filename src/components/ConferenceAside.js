import React from 'react';

export const ConferenceAside = () => (
    <aside className="list conferences aside section">
        <div className="section-inner">
            <h2 className="heading">Conferences</h2>
            <div className="content">
                <ul className="list-unstyled">
                    <li><i className="fa fa-calendar"></i> <a href="https://developer.apple.com/wwdc/" target="_blank">WWDC
                        2014</a> (San Francisco)
                    </li>
                    <li><i className="fa fa-calendar"></i> <a href="http://hive.aigaseattle.org/">Hive</a> (Seattle)
                    </li>
                </ul>
            </div>
        </div>
    </aside>
);

export default ConferenceAside;