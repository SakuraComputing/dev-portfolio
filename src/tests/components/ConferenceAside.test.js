import React from 'react';
import { shallow } from 'enzyme';
import ConferenceAside from '../../components/ConferenceAside';

describe('ConferenceAside Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<ConferenceAside />);
        expect(wrapper).toMatchSnapshot();
    });
});