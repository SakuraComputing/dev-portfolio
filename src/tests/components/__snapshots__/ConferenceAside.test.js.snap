// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConferenceAside Section should render correctly 1`] = `
<aside
  className="list conferences aside section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      Conferences
    </h2>
    <div
      className="content"
    >
      <ul
        className="list-unstyled"
      >
        <li>
          <i
            className="fa fa-calendar"
          />
           
          <a
            href="https://developer.apple.com/wwdc/"
            target="_blank"
          >
            WWDC 2014
          </a>
           (San Francisco)
        </li>
        <li>
          <i
            className="fa fa-calendar"
          />
           
          <a
            href="http://hive.aigaseattle.org/"
          >
            Hive
          </a>
           (Seattle)
        </li>
      </ul>
    </div>
  </div>
</aside>
`;
